tgui/src/interfaces/canister.ract

```diff
@@ -1,5 +1,5 @@
 <ui-notice>
-  <span>The regulator {{data.hasHoldingTank ? "is" : "is not"}} connected to a tank.</span>
+  <span>The regulator {{data.hasHoldingTank ? "is" : "is not"}} is not holding a tank.</span>
```

Comment:
"The regulator is is not holding a tank"
"The regulator is not is not holding a tank"